### DISCUSSION TITLE
Ability to "lock" position of a window

### DISCUSSION THREAD

**Discussion by Xaqaqa** [f1multiviewer/issue-tracker#520] [Feature requests] [answered]
Maybe add a way to lock the stream windows so you don't accidentaly move them about after etting up a layout.
OBS this is a duplicate idea, though the previous idea has not had any response.

**Comment by strice** [comment]
After today's race and constantly having to fiddle with the syncing, even the slightest movement when left clicking in the sync window would move the window. Very annoying. Would love to be able to lock windows/layouts and/or make the left click dragging much less sensitive.

**Comment by LapsTimeOFF** [comment, marked as answer]
Duplicate of #136
